### PR TITLE
update: move Multiplatform journal

### PR DESCRIPTION
**Pull request by koshachy**
* Move and safe delete of all articles
* Delete unused images
* Add redirects to KMP portal

### CODE REVIEW

**Review by zoobestik — APPROVED** [review]

**Comment by zoobestik** [comment]
Target pages isn't merged, but i trust you.

File: redirects/kmm.yml [hunk 1]
```diff
@@ -20,9 +20,6 @@
 - from: /docs/mobile/discover-kmm-project.html
   to: /docs/multiplatform-mobile-understand-project-structure.html
 
-- from: /docs/mobile/introduce-your-team-to-kmm.html
```
Comment (danil-pavlov) [inline redirects/kmm.yml:23]: But now there is no redirect from "/docs/mobile/introduce-your-team-to-kmm.html" to anywhere

File: redirects/pages.yml [hunk 2]
```diff
@@ -770,7 +770,7 @@
 - from:
     - /docs/multiplatform-mobile-introduce-your-team.html
     - /docs/kmm-introduce-your-team.html
-  to: /docs/multiplatform-introduce-your-team.html
```
Comment (danil-pavlov) [inline redirects/pages.yml:773]: The most important redirect is missing, from the last page name "/docs/multiplatform-mobile-introduce-your-team.html" to the new location

File: redirects/kmm.yml (hunk 1 above)
Comment (koshachy) [reply redirects/kmm.yml:23]: https://github.com/JetBrains/kotlin-web-site/pull/4145 fixed here

File: redirects/pages.yml (hunk 2 above)
Comment (koshachy) [reply redirects/pages.yml:773]: https://github.com/JetBrains/kotlin-web-site/pull/4145 fixed here

**Pull request merged**